libs/counterfactuals/src/util/largeCounterfactualsView/getCounterfactualsScatterOption.ts

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { IGenericChartProps, JointDataset } from "@responsible-ai/core-ui";
+
+import { getCounterfactualsScatter } from "./getCounterfactualsScatter";
+
+export function getCounterfactualsScatterOption(
+  xSeries: number[],
+  ySeries: number[],
+  indexSeries: number[],
+  chartProps: IGenericChartProps,
+  jointData: JointDataset,
+  selectedPointsIndexes: number[],
+  customPoints?: Array<{ [key: string]: any }>,
+  isCounterfactualsDataLoading?: boolean,
+  selectPointFromChartLargeData?: (data: any) => void
+): any {
+  const dataSeries = getCounterfactualsScatter(
+    xSeries,
+    ySeries,
+    indexSeries,
+    jointData,
+    selectedPointsIndexes,
+    chartProps,
+    customPoints
+  );
+
+  return {
+    chart: {
+      type: "scatter",
+      zoomType: "xy"
+    },
+    plotOptions: {
+      scatter: {
+        tooltip: {
+          headerFormat: "",
```

Comment:
Is something meant to go here?

Reply:
No, "" is sent. Otherwise highchart displays it's default header format as series 1/n which is not needed by us.